static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -283,25 +271,31 @@ const initialiseEmail = (): void => {
     });
 };
 
-const showFirstVisitBanner = (): void => {
-    if (config.get('switches.idAdConsents', false)) {
-        initFirstPvConsentBanner();
-    } else {
-        initCookiesBanner();
-    }
+const initialiseBanner = (): void => {
+    // ordered by priority
+    const bannerList = [
+        config.get('switches.idAdConsents', false)
+            ? firstPvConsentBanner
+            : cookiesBanner,
+        breakingNews,
+        smartAppBanner,
```

Comment:
As discussed in person, I would check whether we still want to be pointing new mobile web readers to the app